overlay/master_server.py

```diff
@@ -543,11 +583,59 @@ def SubmitValidationJob(self, request: ValidationJobRequest, context):
                     ok = False
                     error_msg = f"{worker_response.hostname} error: {worker_response.error_msg}"
                     errors.append(error_msg)
+                for metric in worker_response.metrics:
+                    flattened_metrics.append(metric)
+
+        if request.spatial_resolution == SpatialResolution.STATE:
+            flattened_metrics = aggregate_metrics_by_state(flattened_metrics)
 
         error_msg = f"errors: {errors}"
         profiler.stop()
 
         return ValidationJobResponse(
+            id=job_id,
+            ok=ok,
+            error_msg=error_msg,
+            duration_sec=profiler.elapsed,
+            metrics=flattened_metrics
+        )
+
+    def SubmitExperiment(self, request: ValidationJobRequest, context) -> ExperimentResponse:
```

Comment:
This what the previous `SubmitValidationJob` was, just moved to a different name so we can still run experiments.